lib/Pipes/PipelineManager.cpp

```diff
@@ -133,6 +137,9 @@ checkComponentsVersion(const revng::DirectoryPath &ExecutionDirectory,
       return Error;
   }
 
+  if (auto Error = TheRunner.purge(ExecutionDirectory))
```

Comment:
I think the code after this if needs to be remove, as it's already part of `purge`

Reply:
You're right, it was forgotten there. Now it's removed.